routes/orders.js

```diff
@@ -0,0 +1,69 @@
+const ordersRouter = require("express").Router();
+const { getAllOrders, getCartByUser, createOrder, updateOrder } = require("../db")
+const { isLoggedIn, isAdmin } = require("./util")
+
+ordersRouter.get("/", isLoggedIn, isAdmin, async (req, res, next) => {
+    try {
+        const allOrders = await getAllOrders();
+        res.send(allOrders);
+    } catch (error) {
+        throw error;
+    };
+});
+
+ordersRouter.get("/cart", isLoggedIn, async (req, res, next) => {
+    const { id } = req.body
+    try {
+        const userCart = await getCartByUser({id});
```

Comment:
if this function is updated to remove the unnecessary curlies, just need to remove them in all calls, too.